care/facility/api/serializers/hospital_doctor.py

```
@@ -8,11 +8,22 @@
         read_only_fields = (
             "id",
             "area_text",
+            "total_doctors"
         )
         exclude = (*TIMESTAMP_FIELDS, "facility", "external_id")
+
+        def to_representation(self, instance):
+            representation = super().to_representation(instance)
+            representation["total_doctors"] = (
+                HospitalDoctors.objects.filter(facility=instance.facility)
+                .aggregate(total_doctors=Sum("count"))["total_doctors"]
```

Ruff (0.8.0)

Undefined name `Sum`

(F821)